tf_encrypted/protocol/pond.py

```diff
@@ -3377,33 +3387,33 @@ def _mask_private(prot: Pond, x: PondPrivateTensor) -> PondMaskedTensor:
     x0, x1 = x.unwrapped
 
     with tf.name_scope("mask"):
-
         with tf.device(prot.crypto_producer.device_name):
-            a = x.backing_dtype.sample_uniform(x.shape)
-            a0, a1 = prot._share(a)
+            a0seed = seed()
+            a1seed = seed()
+
+            a0 = x.backing_dtype.sample_uniform(x.shape, a0seed)
```

Comment:
why `sample_uniform` in one place but `seeded_tensor` in the other?

Reply:
Yeah I think there's something wrong about my implementation!

Reply:
This should just be seeded_tensor. This will also fix some of the naming issues.